app/models/tag.rb

```diff
@@ -23,6 +23,8 @@ class Tag < ActsAsTaggableOn::Tag
   mount_uploader :profile_image, ProfileImageUploader
   mount_uploader :social_image, ProfileImageUploader
 
+  validates :name,
+            format: /\A[A-Za-z0-9]+\z/, allow_nil: true
```

Comment:
TBH - I'm not sure `allow_nil: true` makes much sense to me here. However, I decided to follow the convention of the other validations and felt that _not_ having a not-null constraint for `Tag.name` in the database meant that I probably shouldn't do `allow_nil: false`here. LMK if I'm mistaken!